release.sh

```diff
@@ -301,6 +302,31 @@ function build_from_source() {
   if [[ $? -ne 0 ]]; then
     abort "error building the release"
   fi
+  sign_release
+  # Do not use `||` above or any error will be swallowed.
+  if [[ $? -ne 0 ]]; then
+    abort "error signing the release"
+  fi
+}
+
+# Build a release from source.
+function sign_release() {
+  ## Sign the images with cosign
+  ## For now, check if ko has created imagerefs.txt file. In the future, missing image refs will break
+  ## the release for all jobs that publish images.
+  if [[ -f "imagerefs.txt"  && -z ${SIGNING_KEY} ]]; then
+      echo "Signing Images with the key ${SIGNING_KEY}"
+      COSIGN_EXPERIMENTAL=1 cosign sign $(cat image.refs) --key gcpkms://"${SIGNING_KEY}"
```

Comment:
Where does this output go / get stored?

I see the next output being redirected to a file and saved.

Reply:
Is `image.refs` supposed to be the same string as `imagerefs.txt`?

Reply:
Running `cosign sign` signs the image and pushes it to the registry.



> Is `image.refs` supposed to be the same string as `imagerefs.txt`?

Yup, typo.